apps/front-end/src/components/panel/directoryPanel/DirectoryPanel.tsx

```diff
@@ -25,12 +24,12 @@ const StyledDirectoryPanel = styled(Box)(() => ({
 }));
 
 const fetchResults = async () => {
-  const response = await searchDataset({
-    params: { datasetId: "delhi" },
-    query: {
-      returnProps: ["country_id"],
-    },
-  });
+  const datasetId = getDatasetId();
+  if (datasetId === null) {
```

Comment:
I don't entirely trust `datasetId` won't be `undefined` here, because TS seems to have a bit of a blind spot for it...